T/TurboPFor/build_tarballs.jl

```diff
@@ -0,0 +1,61 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "TurboPFor"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/powturbo/TurboPFor-Integer-Compression.git", "43fb0b2abaef27f6753f4494ffff638c3002f24c")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd TurboPFor-Integer-Compression/
+make -j${nprocs}
+cd ..
+cp -r TurboPFor-Integer-Compression $WORKSPACE/destdir/
```

Comment:
I don't think it's useful to dump the content of this directory in the prefix. All those object files are useless for our purposes, better to install only one what you need

Reply:
Ok thanks for pointing that out! So I should just delete these last 2 lines? I did it because BB couldn't find the executables otherwise.

Reply:
Just install what you actually need